src/runtime.js

```diff
@@ -0,0 +1,38 @@
+;(function(){
+
+  function Runtime(context){
+    this.context = context;
+  };
+
+  function BrowserRuntime(context){
+    this.context = context;
+  }
+
+  function NonBrowserRuntime(context){
+    this.context = context;
+  }
+
+  Runtime.prototype.getWindow = function() {
+    return this.context;
+  };
+
+  BrowserRuntime.prototype = new Runtime();
+  BrowserRuntime.prototype.constructor = BrowserRuntime;
```

Comment:
We have a `Pusher.Util` function for doing inheritance. Maybe this was done here because of the loading order? For consistency we should try to use the existing function.